doc/help/kdb-list-commands.md

```diff
@@ -2,13 +2,13 @@
 
 ## SYNOPSIS
 
-`kdb list-commands`
+`kdb list-commands [OPTION]...`
```

Comment:
This is inconsistent to the other man pages. We should either have `[OPTIONS]` everywhere or nowhere. The 3 dots at the end are misleading (there is not any other argument taken here).

Reply:
How man pages work is clearly defined in [man(1)](http://man7.org/linux/man-pages/man1/man.1.html). It says, i.a.

> The command or function illustration is a pattern that should match all possible invocations.

What is currently in master does not do that, thus I added it.

>  [expression] ...   entire expression within [ ] is repeatable

And as far as I can tell using multiple options is possible: `kdb list-commands -C always -v -0`, thus using 3 dots looks correct to me.

My proposition adheres to the synopsis of GNU coreutils like [chown](http://man7.org/linux/man-pages/man1/chown.1.html), [chmod](http://man7.org/linux/man-pages/man1/chmod.1.html) and [ls](http://man7.org/linux/man-pages/man1/ls.1.html) and I think that we should apply this to other man pages, too.

Reply:
It is unfortunately not consistently done across different tools, even not within the man tools themselves e.g. `whereis` no dots, `apropos` lists the possible options, ....

> thus using 3 dots looks correct to me.

rsync uses `[OPTION...]`

But this is actually not relevant, I was only complaining that we should consistent **within ourselves**. And we were consistent before, and this PR was the first to add `[OPTION]...`.

So either we add it everywhere, or leave it as is (which sounds to me the better way, as it is not really an important thing).

Reply:
@Chemin1 could you please rebase and fix this little thing. It would be great to merge this work.